lib/postfix_version.rb

```diff
@@ -0,0 +1,7 @@
+if  FileTest.exists?("/etc/postfix/main.cf")
```

Comment:
I'd rather have this condition instead the `setcode` block

Reply:
Sorry, i don't exactly know what you mean?